shell/browser/ui/devtools_manager_delegate.cc

```diff
@@ -27,9 +27,13 @@
 #include "net/base/net_errors.h"
 #include "net/socket/stream_socket.h"
 #include "net/socket/tcp_server_socket.h"
+#include "shell/browser/browser.h"
 #include "shell/browser/electron_paths.h"
+#include "third_party/inspector_protocol/crdtp/dispatch.h"
 #include "ui/base/resource/resource_bundle.h"
 
+const char kBrowserCloseMethod[] = "Browser.close";
```

Comment:
can you move this to the anonymous namespace below

Reply:
Done